pkg/openthread/include/ot.h

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (C) 2017 Fundacion Inria Chile
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    pkg_openthread_cli   OpenThread
+ * @ingroup     pkg_openthread
+ * @brief       An open source implementation of Thread stack
+ * @see         https://github.com/openthread/openthread
+ *
+ * Thread if a mesh oriented network stack running for IEEE802.15.4 networks.
+ * @{
+ *
+ * @file
+ *
+ * @author  Jos� Ignacio Alamos <jialamos@uc.cl>
+ */
+
+#ifndef OT_H
+#define OT_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "net/netopt.h"
+#include "net/ieee802154.h"
+#include "net/ethernet.h"
+#include "net/gnrc/netdev.h"
+#include "thread.h"
+#include "openthread/types.h"
+
+#define OPENTHREAD_XTIMER_MSG_TYPE_EVENT (0x2235)   /**< xtimer message receiver event*/
+#define OPENTHREAD_NETDEV_MSG_TYPE_EVENT (0x2236)   /**< message received from driver */
+#define OPENTHREAD_SERIAL_MSG_TYPE_EVENT (0x2237)   /**< event indicating a serial (UART) message was sent to OpenThread */
+#define OPENTHREAD_MSG_TYPE_RECV (0x2238)           /**< event for frame reception */
```

Comment:
this is very weird... I have them alligned.
It seems to be a different file. I'm checking